packages/turbo/android/src/main/java/com/reactnativeturbowebview/RNVisitableViewManager.kt

```diff
@@ -15,7 +15,9 @@ enum class RNVisitableViewEvent(val jsCallbackName: String) {
   WEB_ALERT("onWebAlert"),
   WEB_CONFIRM("onWebConfirm"),
   FORM_SUBMISSION_STARTED("onFormSubmissionStarted"),
-  FORM_SUBMISSION_FINISHED("onFormSubmissionFinished")
+  FORM_SUBMISSION_FINISHED("onFormSubmissionFinished"),
+  SHOW_VISITABLE_ACTIVITY_INDICATOR("onShowVisitableActivityIndicator"),
+  HIDE_VISITABLE_ACTIVITY_INDICATOR("onHideVisitableActivityIndicator")
```

Comment:
I think the method names could be improved here. Do we need `Visitable` as part of the name?
There's also a disconnect between `renderLoading` prop name and `..ActivityIndicator..`.

What about `onShowLoading` and on `onHideLoading`?